### PR TITLE
Issue65 validate all examples

### PR DESCRIPTION
**Pull request by RalphTro**
Hi @Echsecutor , 
ALL XML events and all but one JSON/JSON-LD files (--> came across two things to be discussed with José/Mark - so this effort was also valuable for the standardisation work) are now conformant to the respective schemas (still WIP though). Just in case you would like to go on...
I will adjust the file in question (epcisDocWithXMLstartTagAndErrorDeclaration.jsonld) as soon as the schema is finalised. 
Kind regards,
@RalphTro

### CODE REVIEW

File: tests/examples/ReferenceEventHashAlgorithm3.jsonld [hunk 1]
```diff
@@ -1,9 +1,14 @@
 {
-  "@context": ["https://gs1.github.io/EPCIS/epcis-context.jsonld",{"example": "https://ns.example.com/epcis"}],
```
Comment (Echsecutor) [inline tests/examples/ReferenceEventHashAlgorithm3.jsonld:2]: example namespace/context is needed below

File: tests/examples/epcisDocWithDefaultSchemaInContext.jsonld [hunk 2]
```diff
@@ -1,42 +1,67 @@
...
-            "bizTransactionList": [ 
-                     {"type": "urn:epcglobal:cbv:btt:po", "bizTransaction": "http://transaction.acme.com/po/12345678" },
-                     {"type": "urn:epcglobal:cbv:btt:desadv", "bizTransaction": "urn:epcglobal:cbv:bt:0614141073467:1152" }
-                    ]
-       }
-     ]
+  "@context": [
+    "https://gs1.github.io/EPCIS/epcis-context.jsonld",
+    {
+      "example": "http://ns.example.com/epcis/"
```
Comment (Echsecutor) [inline tests/examples/epcisDocWithDefaultSchemaInContext.jsonld:5]: example context probably does not hurt here, but is not needed?

File: tests/examples/epcisDocWithSingleEvent.jsonld [hunk 3]
```diff
@@ -2,34 +2,36 @@
   "@context": [
     "https://gs1.github.io/EPCIS/epcis-context.jsonld",
     {
-      "example": "http://ns.example.com/epcis/"
+      "gs1": "https://gs1.org/voc/"
```
Comment (Echsecutor) [inline tests/examples/epcisDocWithSingleEvent.jsonld:5]: probably we need neither example nor gs1 context in this example, but does not hurt to keep it

**Review by Echsecutor — CHANGES_REQUESTED** [review]
I did not comment on every place where a gs1 context is added abut not needed, those actually don't matter. But there is one place where we do need the example context, otherwise we can not parse the jsonld file

**Comment by RalphTro** [comment]
Hi @Echsecutor, 
Made quite a number of adjustments of namespace declarations (I am not 100% sure if esp. the json files are already correct - the current version of the JSON schema does not seem to take ns declarations into consideration yet (alerted José to this) and especially replaced inline declarations with header-based one. Additional/superflous ns decalarations can IMHO be disregarded, so we can leave them in.

**Comment by Echsecutor** [comment]
fixes https://github.com/RalphTro/epcis-event-hash-generator/issues/37

**Review by Echsecutor — APPROVED** [review]

**Pull request merged**